src/views/Project/bugs.graphql

```diff
@@ -0,0 +1,16 @@
+fragment Bugs on BugPager{
+      edges {
+          node {
+            assignedTo {
+              name
+            }
+            component
+            lastChanged
+            id
+            summary
+            status
+            resolution
+            keywords
+          }
+    }
+}
```

Comment:
New line at end of file. Also looks like the indentation and spacing may have been messed up in this file.